etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/orders/OrderControllerTest.groovy

```diff
@@ -12,14 +12,14 @@ import org.hl7.fhir.r4.model.Bundle
 import spock.lang.Specification
 
 class OrderControllerTest extends Specification {
-    def ruleEngine = Mock(RuleEngine)
+    def ruleEngine = Mock(ValidationRuleEngine)
 
     def setup() {
         TestApplicationContext.reset()
         TestApplicationContext.init()
         TestApplicationContext.register(OrderController, OrderController.getInstance())
         TestApplicationContext.register(MetricMetadata, Mock(MetricMetadata))
-        TestApplicationContext.register(RuleEngine, ruleEngine)
+        TestApplicationContext.register(ValidationRuleEngine, ruleEngine)
```

Comment:
Same thing here, I think it should be RuleEngine for the register

Reply:
Same as above